### ISSUE TITLE
Randomly getting broken spear thrown to get back in the slot where it was thrown from.

### ISSUE THREAD

**Issue by valipopescu** [anegostudios/VintageStory-Issues#670]
**Game Version:** 1.14.0 rc-4
**Platform:** Windows 
**Modded:** No 

### Description
The flint spear that nearly breaks (should break on a throw) does so as normal but you get it back as if nothing happened. The damage is applied as normal. Doesn't happen when there's a replacement spear in inventory. Can not get it to do that consistently, although if it's not a bug... would appreciate some consistency from it :D (probably is a bug related to how the application of durability loss is done on throwing )

### How to reproduce
1. Make a spear get it to nearly breaking level (3 durability left).
2. Use it on throwing at a rabbit
3. It breaks as normal but then you randomly get it back in your inventory as if it never cracked. Can be thrown again ... Managed to shoot one 12 times :)

### Expected behavior
I didn't expect it to come back to inventory but if you think this bug is a feature I'm more than happy if you make it consistent :D 

### Screenshots
Attach a screenshot if it helps explain the issue / shows a visual bug.
I didn't take screenshots nor was I streaming when I was playing .

### Logs
Basically nothing in the logs.

**Comment by Craluminum2413** [comment]
So this is duplicate of #386 and also applies to multiplayer label

**Comment by copygirl** [comment]
Yeah I think this is the same issue.
